cmd/agent/daemon/state/netflow_pipeline.go

```diff
@@ -53,64 +53,35 @@ func (c *Controller) runNetflowPipeline(ctx context.Context) error {
 	}
 	c.log.Infof("fetched cluster info, pod_cidr=%s, cluster_cidr=%s", c.clusterInfo.podCidr, c.clusterInfo.serviceCidr)
 
-	groupFlows := c.cfg.NetflowGrouping != 0
-
+	lastExportedAt := time.Now().UTC()
 	errg, ctx := errgroup.WithContext(ctx)
 	errg.Go(func() error {
 		for {
 			select {
 			case <-ctx.Done():
 				return ctx.Err()
 			case e := <-c.tracer.NetflowEvents():
```

Comment:
Probably nitpick: 
After thinking about this, we might want to add a timer to flush the netflows from time to time. Otherwise we might end up not reporting any netflows for workloads that only perform network actions before we hit the first `c.cfg.NetflowExportInterval`.

Reply:
I'm thinking that is the chance that you will not have any network activity for a long period of time for any container on the node. If there is at least one long lived connection we still receive samples here and it will trigger export.